src/main/java/gift/entity/Product.java

```diff
@@ -0,0 +1,43 @@
+package gift.entity;
+
+public class Product {
+    private Long id;
+    private String name;
+    private Long price;
+    private String imageUrl;
+
+    public Product(Long id, String name, Long price, String imageUrl) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+        this.imageUrl = imageUrl;
+    }
+
+    // Getter
+    public Long getId() {
+        return id;
+    }
+    public String getName() {
+        return name;
+    }
+    public Long getPrice() {
+        return price;
+    }
+    public String getImageUrl() {
+        return imageUrl;
+    }
```

Comment:
메서드와 메서드 사이에는 줄바꿈을 통해서 메서드의 가독성을 높여주시면 좋을 것 같아요. 😃 

```suggestion
    public Long getId() {
        return id;
    }
    
    public String getName() {
        return name;
    }
    
    public Long getPrice() {
        return price;
    }
    
    public String getImageUrl() {
        return imageUrl;
    }
```